src/adapters/refract/transformSections.coffee

```diff
@@ -2,37 +2,65 @@ _ = require('./helper')
 
 blueprintApi = require('../../blueprint-api')
 getDescription = require('./getDescription')
+
 transformResources = require('./transformResources')
+transformResource = require('./transformResource')
+
 
-module.exports = (element) ->
+module.exports = (parentElement) ->
   resourceGroups = []
-  resourceGroupsElements = _.chain(element)
-                    .content()
-                    .filter({element: 'category', meta: {classes: ['resourceGroup']}})
-                    .value()
-
-  # If we can't find any resource group we can assume that there are Resources directly
-  if resourceGroupsElements.length is 0
-    resourceGroups.push(
-      new blueprintApi.Section({
-        name: ''
+
+  # List of Application AST Resources (= already transformed
+  # Refract Resources into the Application AST Resource).
+  applicationAstResources = []
+
+  _.forEach(parentElement.content, (element, index) ->
+    # If the element is a resource, transform it
+    # into the Application AST Resource and push it
+    # to a temporary array, we'll create an artificial
+    # resource group (section) for these resources
+    # later in the process.
```

Comment:
Nit: I mentioned below comments that say what is being done. This comment looks like it is just English for the code below.